storybook/index.js

```diff
@@ -0,0 +1,19 @@
+import { AppRegistry } from 'react-native';
+import { getStorybookUI, configure } from '@storybook/react-native';
+
+import './rn-addons';
+
+// import stories
+configure(() => {
+  require('./stories');
+}, module);
+
+// Refer to https://github.com/storybookjs/storybook/tree/master/app/react-native#start-command-parameters
+// To find allowed options for getStorybookUI
+const StorybookUIRoot = getStorybookUI({});
+
+// If you are using React Native vanilla and after installation you don't see your app name here, write it manually.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 116. Maximum allowed is 100.](https://app.codacy.com/gh/nezago/phone-history-store-frontend/pullRequest?prid=5545949)